### ISSUE TITLE
Overwriting profile is not possible

### ISSUE THREAD

**Issue by kleinesfilmroellchen** [Kobzol/cargo-pgo#37]
While I can use `--profile=xyz` to pass a different profile to Cargo, this conflicts with the `--release` option that cargo-pgo always supplies. In my case, since other tools like wasm-pack are also inflexible in terms of Cargo profiles, I cannot use `--release` and have (multiple) separate release profiles for different purposes.

Therefore, it would be nice to have a profile option on cargo-pgo directly, which will override the `--release` argument for Cargo. The default doesn't have to be changed.

**Comment by Kobzol** [comment]
Hi! Good point, that was an oversight on my part. Can you please check if this branch solves your issue?

```bash
$ cargo install --git https://github.com/kobzol/cargo-pgo --branch examine-profile
```

**Comment by hatoo** [comment]
@Kobzol 
It works in my environment.
Could you release this feature?

**Comment by Kobzol** [comment]
https://github.com/Kobzol/cargo-pgo/pull/45 should fix this, I will publish a new release shortly.

**Comment by Kobzol** [comment]
Published `v0.2.6` with the fix (and a few other fixes).
